src/collections/_documentation/accounts/quotas/manage-event-stream-guide.md

```diff
@@ -3,30 +3,30 @@ title: Manage Your Event Stream, a Guide
 sidebar_order: 3
 ---
 
-Getting started with Sentry error monitoring is pretty straightforward --- create a Sentry project, install and initialize the Sentry SDK in your code and you're ready to go. Once completed, **any** Exception, Runtime Error or Crash that occurs **wherever** and **whenever** your code is running, will be aggregated in your `Event Stream`. Depending on your portfolio, your stream might be aggregating millions of events in no time. Sending all your errors to Sentry is generally a good practice. This ensures greater visibility into the health and performance of your application code. However, with just this basic setup in place, you might risk:
+Sending all of your application errors and crashes to Sentry ensures that you'll be notified in real-time as errors happen in your code. However, with just the basic setup in place, you might soon realize that your applications are generating a lot of errors, and those might trigger quite a few notifications. Sentry provides various tools that give you control over the _type_ and _amount_ of errors that are monitored. These will allow you to:
```

Comment:
Suggest (to minimize wordiness): "Sending all application events to Sentry ensures you'll be notified in real-time when errors occur in your code. However, if you have only the basic setup for notifications, you may realize quickly that your applications generate many errors, triggering many notifications. Sentry provides tools to control the _type_ and _amount_ of errors that are monitored. These allow you to:"